content/en/project/Detecting-ADHD-through-fMRI-signals-using-ML-classification-models/index.md

```diff
@@ -0,0 +1,237 @@
+---
+type: "project" # DON'T TOUCH THIS ! :)
+date: "2024-06-14" # Date you first upload your project.
+# Title of your project 
+title: "Detecting ADHD through fMRI signals using ML classification models"
+
+# List the names of the collaborators within the [ ].
+names: [Agustín Rodriguez, Ezequiel Sirne, Francisco Covelli]
+
+# Your project GitHub repository URL
+github_repo: https://github.com/AgustinNR/BrainHack-BsAs-Humai-Team1
+
+# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/PSY6983-2021/project_template), click `manage topics`.
+# Please only lowercase letters
+tags: [adhd, fmri, machine-learning, rs-fmri, xgboost, PCA, classification, bold-signal, connectivity]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects..
+summary: "We used the ADHD-200 Sample dataset to implement various machine learning classification models, aimed at diagnosing ADHD through resting-state fMRI signals."
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: "fotor-ai-20240613194347.jpg"
+---
+
+<img src="fotor-ai-20240613194347.jpg" alt="Cover Image" width="500"/>
```

Comment:
You should remove this image: the cover is automatically added at the start of the project, you don't need to do it yourself.
```suggestion
```

Reply:
Thank you for the advice! Please find the corrected index.md file pushed.